<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioFileToTriggerParamsWidget</class>
 <widget class="QWidget" name="AudioFileToTriggerParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_actionCB">
       <item>
        <property name="text">
         <string>Start</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pause</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Resume</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="m_dependentWidgetsParent" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="m_maxDistanceDSB">
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Volume (dB)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Attenuation rolloff factor</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min. distance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>-96</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="m_volumeSlider">
             <property name="minimum">
              <number>-96</number>
             </property>
             <property name="maximum">
              <number>12</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSpinBox" name="m_volumeSB">
           <property name="minimum">
            <number>-96</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="m_attenuationRolloffFactorDSB">
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="m_loopingChB">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Positional</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="m_attenuationModeCB">
        <item>
         <property name="text">
          <string>No attenuation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inverse distance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear distance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exponential distance</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Max. distance</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Attenuation mode</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="m_positionalChB">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Looping</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="m_minDistanceDSB">
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_volumeSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_volumeSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_volumeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
